test/browser/browser_asrouter_targeting.js

```diff
@@ -110,6 +110,16 @@ add_task(async function checkProfileAgeReset() {
     "should select correct item by profile age reset");
 });
 
+add_task(async function checkCurrentTimestamp() {
+  let message = {id: "foo", targeting: `currentTimestamp < '${new Date(new Date().getTime() + 1000)}'|date`};
```

Comment:
you should be able to do `new Date(Date.now() + 1000)` here instead of nesting the new Date's again